migrations/0014_media.sql

```
@@ -0,0 +1,296 @@
+	session_id TEXT REFERENCES project_sessions (id)
+);
+
+INSERT INTO generations_new (
+	id, user_id, result_media_id, source_media_id, prompt, kind, amount,
+	balance_after, created_at, session_id
+)
+SELECT
+	g.id,
+	g.user_id,
+	result_media.media_id,
+	source_media.media_id,
```

SQLFluff (4.3.0)

[error] Reuse of column alias 'media_id' from line 161.

(AL08)